src/solvers/augmented_Lagrangian_method.jl

```diff
@@ -265,13 +271,39 @@ function augmented_Lagrangian_method(
     h=nothing,
     grad_g=nothing,
     grad_h=nothing,
+    inequality_constrains=-1,
+    equality_constrains=-1,
```

Comment:
Are these arguments documented? Ctrl+F in browser doesn't indicate so.

Reply:
Works even a bit better with nothings :)